chapter2_mdw/ExerciseThree

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/perl
+
+use 5.010;
+use warnings;
+
+chomp($radius = <STDIN>);
+$pi = 3.141592654;
+
+$circumference = 2*$pi*$radius;
+
+if ($radius <= 0){
```

Comment:
If I was going to be really critical, the requirement is "if the user enters a number less than zero", not less than or equal to. :smile: